joml-geometry/src/main/java/org/terasology/joml/geom/AABBf.java

```diff
@@ -130,6 +130,117 @@ public AABBf set(AABBfc source){
         return this;
     }
 
+    /**
+     * Set {@link AABBf} at x,y,z with a size of zero.
+     *
+     * @param pos position of the center
+     * @return this
+     */
+    public AABBf set(Vector3dc pos){
+        this.minX = (float) pos.x();
+        this.minY = (float) pos.y();
+        this.minZ = (float) pos.z();
+        this.maxX = (float) pos.x();
+        this.maxY = (float) pos.y();
+        this.maxZ = (float) pos.z();
+        return this;
+    }
+
+    /**
+     * Set {@link AABBf} at x,y,z with a size of zero.
+     *
+     * @param pos position of the center
+     * @return this
+     */
+    public AABBf set(Vector3fc pos){
+        this.minX = pos.x();
+        this.minY = pos.y();
+        this.minZ = pos.z();
+        this.maxX = pos.x();
+        this.maxY = pos.y();
+        this.maxZ = pos.z();
+        return this;
+    }
+
+    /**
+     * Set {@link AABBf} at x,y,z with a size of zero.
+     *
+     * @param x
+     *          the x coordinate of the minimum corner and maximum corner
+     * @param y
+     *          the y coordinate of the minimum corner and maximum corner
+     * @param z
+     *          the z coordinate of the minimum corner and maximum corner
+     * @return this
+     */
+    public AABBf set(float x, float y, float z){
+        this.minX = x;
+        this.minY = y;
+        this.minZ = z;
+        this.maxX = x;
+        this.maxY = y;
+        this.maxZ = z;
+        return this;
+    }
+
+
+    /**
+     * Create a new {@link AABBf} at x,y,z with a size of zero.
+     *
+     * Can be used with conjuction with {@link #expand(Vector3dc)} to create an aabb around a center and an extent or
+     * {@link #setSize} for an aabb from min to min + size.
+     *
+     * @param pos position of the center
+     */
+    public AABBf(Vector3dc pos) {
```

Comment:
Please add a test for 
```java
new AABBf(pos).containsPoint(pos)
```

to me it is not clear whether this should return true or false... but we should at least make sure that we don't accidentally change the way this works...